recipes/pipefunc/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pipefunc" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pipefunc-{{ version }}.tar.gz
+  sha256: 988ef01177d73cfd8f38c86999ac4819818c562e75aee101b34fc6502877b623
+
+build:
+  noarch: python
```

This should fix Windows:

```suggestion
  noarch: python
  entry_points:
    - pipefunc-launcher = pipefunc._server_support.launcher:main
```

---

Thanks for noticing this. This script was actually removed before v0.1.0. I just released v0.2.0 and fixed this.